### DISCUSSION TITLE
Feature Request: GPS status/condition overlay

### DISCUSSION THREAD

**Discussion by joshellington** [gopro/labs#200] [Feature Requests]
Would love a way either through the existing burn-in overlay preset or any other option to better understand the ongoing GPS satellite lock status and "accuracy".

Currently monitoring the interface icon and/or running the GPMF parser in post makes it difficult to know when and how well the GPS signal is doing while in the field. 

Let me know if I can provide anymore context. Really appreciate y'all's effort to expand the capabilities of your platform!

**Comment by dnewman-gpsw** [comment]
There is an undocumented "feature" for this.  Try the command oMFLAG=1  (and use '!' instead of 'o', to make it survive a power cycles.)

**Comment by KonradIT** [comment]
How come `FLAG` is GPS? Any way to put this on the front screen?

**Comment by dnewman-gpsw** [comment]
FLAG is a bit field of lots of random system status fields, not intended to be user friendly.  Try oMFLAG=127  What is outputs can change between cameras.  

To display also on the front screen HERO10 and 11 is supported with oMFRNT=1. Unfortunately there are some scaling artifacts for the smaller text within the FLAG fields.

**Comment by tranzmatt** [comment]
Is there an API call that will allow me to get the camera's current GPS location?

**Reply by dnewman-gpsw** [reply]
Given that QR codes are write only, I'm not sure how you would use that API. Within hero 11 the new scripting allows for GPS distance from a known point, but there is no independent longitude and latitude variables.
